docs/monitors/kubernetes-controller-manager.md

```diff
@@ -0,0 +1,598 @@
+<!--- GENERATED BY gomplate from scripts/docs/templates/monitor-page.md.tmpl --->
+
+# kubernetes-controller-manager
+
+Monitor Type: `kubernetes-controller-manager` ([Source](https://github.com/signalfx/signalfx-agent/tree/master/internal/monitors/kubernetes/controllermanager))
+
+**Accepts Endpoints**: **Yes**
+
+**Multiple Instances Allowed**: Yes
+
+## Overview
+
+Exports Prometheus metrics from the [kube-controller-manager]
+(https://kubernetes.io/docs/reference/command-line-tools-reference/kube-controller-manager/).
+The monitor queries path `/metrics` by default when no path is configured. The monitor converts
+the Prometheus metric types to SignalFx metric types as described [here](prometheus-exporter.md)
+
+An example configuration that pulls metrics from the HTTP endpoint.
+
+```yaml
+monitors:
+- type: kubernetes-proxy
+  host: localhost
```

Comment:
Show it with a discoveryRule.

Reply:
Updated to use a discoveryRule. This depends on #907